src/main/java/com/google/api/codegen/config/GapicMethodConfig.java

```diff
@@ -277,6 +271,17 @@ static ResourceNameTreatment defaultResourceNameTreatment(
     return defaultResourceNameTreatment;
   }
 
+  @VisibleForTesting
```

Comment:
Are you testing this directly? If not, we can remove this annotation.

Reply:
Yes, this is being tested a couple times in [ResourceNameMessageConfigsTest.java](https://github.com/googleapis/gapic-generator/blob/3f92b3c11fe4017e819a262a66ce4d6120dba94b/src/test/java/com/google/api/codegen/config/ResourceNameMessageConfigsTest.java#L469)

Reply:
Ah ok. So that means that the new overload (on line 244) is not being tested, and can be private?